components/clp-package-utils/clp_package_utils/scripts/decompress.py

```diff
@@ -218,6 +208,12 @@ def handle_extract_stream_cmd(
     # fmt: on
 
     if EXTRACT_IR_CMD == job_command:
+        if hasattr(parsed_args, "dataset"):
```

Comment:
Do we need to check this?
My understanding is that since we don;t explicilty add dataset when target is IR, the script will return an error if we try to specify --dataset the cmdline